exts/cesium.usd.plugins/cesium/usd/plugins/CesiumUsdSchemas/__init__.pyi

```diff
@@ -5,22 +5,27 @@ __all__ = [
     "Data",
     "Georeference",
     "Imagery",
-    "TilesetAPI",
+    "Tileset",
     "Tokens",
 ]
 
+
 # TODO: Fill out these stubs since we can't seem to autogen them.
 class Data(pxr.Usd.Typed, pxr.Usd.SchemaBase, Boost.Python.instance):
     pass
 
+
 class Georeference(pxr.Usd.Typed, pxr.Usd.SchemaBase, Boost.Python.instance):
     pass
 
+
 class Imagery(pxr.Usd.Typed, pxr.Usd.SchemaBase, Boost.Python.instance):
     pass
 
-class TilesetAPI(pxr.Usd.APISchemaBase, pxr.Usd.SchemaBase, Boost.Python.instance):
+
+class Tileset(pxr.Usd.Typed, pxr.Usd.SchemaBase, Boost.Python.instance):
     pass
```

Comment:
Could you also add `Session` to this file? I forgot to do that in https://github.com/CesiumGS/cesium-omniverse/pull/295

Reply:
Ah, apologies. I missed the note on the TODO I wrote to add that. Added.